src/metabase/api/session.clj

```diff
@@ -58,6 +59,8 @@
     (events/publish-event! :user-login
       {:user_id (u/the-id user), :session_id (str session-uuid), :first_login (nil? (:last_login user))})
     (record-login-history! session-uuid (u/the-id user) device-info)
+    (when (nil? (:last_login user))
```

Comment:
Is `:last_login` ever going to be `false`? No, so you can just use `when-not` here, right? 

The only reason we're doing `(nil? (:last_login user))` in the `publish-event!` line above is because we literally want a true-or-false boolean for that event